src/firebase/firebaseConfig.ts

```
@@ -0,0 +1,23 @@
+import { initializeApp, getApps } from 'firebase/app';
```

ESLint

[error] initializeApp not found in 'firebase/app'

(import/named)